.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow installs the package and builds the docs
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: build the docs
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  pytest:
```

Comment:
This shows up in the output, so maybe `sphinx:`?

Reply:
done